test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/lang/SystemTest.scala

```diff
@@ -147,4 +148,10 @@ class SystemTest {
       assertEquals(().hashCode(), System.identityHashCode(()))
     }
   }
+
+  @Test def getenv_should_return_an_unmodifiable_map(): Unit = {
+    assertTrue(System.getenv().isInstanceOf[java.util.Map[String, String]])
+
+    expectThrows(classOf[Exception], System.getenv.put("", ""))
+  }
 }
```

Comment:
this could be a problem ... depends on the presence of ```ANYVAR``` variable present in the environment when running tests on the JVM.

Should we do this assumption?